### PR TITLE
Fixes PFN CI tests

### PR DESCRIPTION
**Pull request by prabhatnagarajan**
This fixes the build and changes the supported Python version to 3.6.

### CODE REVIEW

File: examples/grasping/train_dqn_batch_grasping.py [hunk 1]
```diff
@@ -206,7 +206,6 @@ def make_env(idx, test):
             maxSteps=max_episode_steps,
             isTest=test,
         )
-        assert env.observation_space is None
```
Comment (prabhatnagarajan) [inline examples/grasping/train_dqn_batch_grasping.py:209]: I removed this line after this recent update to the environment: https://github.com/bulletphysics/bullet3/commit/3a6a279b9a757c4c1306fc55fa6e95df40c89574#diff-37edfb1b0750fd6a650da86f2d5c1ec3827623de1abb22ba2fc7cf897eeb5c44

File: examples/atari/reproduction/a3c/train_a3c.py [hunk 2]
```diff
@@ -1,9 +1,6 @@
 import argparse
 import os
 
-# Prevent numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/atari/reproduction/a3c/train_a3c.py:6]: You should not move this line. This must be done before `import numpy`.

File: examples/atari/train_a3c_ale.py [hunk 3]
```diff
@@ -1,9 +1,6 @@
 import argparse
 import os
 
-# Prevent numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/atari/train_a3c_ale.py:6]: Ditto

File: examples/atari/train_acer_ale.py [hunk 4]
```diff
@@ -1,9 +1,6 @@
 import argparse
 import os
 
-# Prevent numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/atari/train_acer_ale.py:6]: Ditto

File: examples/atari/train_nsq_ale.py [hunk 5]
```diff
@@ -2,9 +2,6 @@
 import os
 import random
 
-# This prevents numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/atari/train_nsq_ale.py:7]: Ditto

File: examples/gym/train_a3c_gym.py [hunk 6]
```diff
@@ -12,9 +12,6 @@
 import argparse
 import os
 
-# This prevents numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/gym/train_a3c_gym.py:17]: Ditto

File: examples/gym/train_acer_gym.py [hunk 7]
```diff
@@ -1,9 +1,6 @@
 import argparse
 import os
 
-# This prevents numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/gym/train_acer_gym.py:6]: Ditto

File: examples/gym/train_pcl_gym.py [hunk 8]
```diff
@@ -12,9 +12,6 @@
 import argparse
 import os
 
-# This prevents numpy from using multiple threads
-os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
-
```
Comment (muupan) [inline examples/gym/train_pcl_gym.py:17]: Ditto

**Review by muupan — CHANGES_REQUESTED** [review]

**Comment by muupan** [comment]
Can you explain why it needs python 3.6?

**Comment by prabhatnagarajan** [comment]
Some of the previous PR CIs were failing:
https://ci.preferred.jp/chainerrl.py3.cpu/62915/
```
2020-09-16 16:48:46.403755 STDOUT 2092] | Downloading https://files.pythonhosted.org/packages/53/10/776750d57ade26522478a92a2e14035868624a6a62f4157b0cc5abd4a980/scipy-1.5.2.tar.gz (25.4MB) | 5.1s
-- | -- | --
2020-09-16 16:48:51.473617 STDOUT 2092] | Complete output from command python setup.py egg_info: |  
2020-09-16 16:48:51.473630 STDOUT 2092] | Traceback (most recent call last): |  
2020-09-16 16:48:51.473644 STDOUT 2092] | File "<string>", line 1, in <module> |  
2020-09-16 16:48:51.473747 STDOUT 2092] | File "/tmp/pip-build-sxosch92/scipy/setup.py", line 31, in <module> |  
2020-09-16 16:48:51.473749 STDOUT 2092] | raise RuntimeError("Python version >= 3.6 required.") |  
2020-09-16 16:48:51.473756 STDOUT 2092] | RuntimeError: Python version >= 3.6 required.
```

I thought it would be ok to update the required version to 3.6 (at least for testing).

**Review by muupan — APPROVED** [review]
While E402 can be addressed in a different way (https://gitlab.com/pycqa/flake8/-/issues/638), I think it is good to merge as it is since fixing the CI asap is important.

**Pull request merged**